src/v2/Apps/Categories/Components/CategoriesIntro.tsx

```diff
@@ -6,7 +6,9 @@ export const CategoriesIntro: React.FC = () => {
     <>
       <GridColumns>
         <Column span={6}>
-          <Text variant="xl">The Art Genome Project</Text>
+          <Text variant="xl">
+            <h1>The Art Genome Project</h1>
+          </Text>
```

Comment:
Could use `as` to avoid the extra nested markup here I believe? `<Text as="h1" variant="xl">…</Text>`